src/Vlingo.Http/RequestParser.cs

```diff
@@ -221,7 +223,7 @@ private string Compact()
             private Optional<string> NextLine(string errorResult, string errorMessage)
             {
                 var possibleCarriageReturnIndex = -1;
-                var lineBreak = _requestText.IndexOf("\n", _position, StringComparison.InvariantCultureIgnoreCase);
+                var lineBreak = _requestText.IndexOf(NewLineDelimiter, _position);
```

Comment:
`IndexOf` supports both, string and char so maybe the difference is on your local culture and the fact that `StringComparison.InvariantCultureIgnoreCase` was used.